published/AWS/servers_by_account_and_tags/function.js

```diff
@@ -0,0 +1,119 @@
+function(account_label, env_tag_value, ellipsis) {
+  "use strict";
+const Q = require('q');
+var AWS = require('aws-sdk');
+
+// -----------------------------------------------------------------------------------------------
+// INPUTS
+// -----------------------------------------------------------------------------------------------
+
+// Add the region codes for the regions you want Ellipsis to search for instances
+var regions = [
+  'us-east-1',
+  'us-west-1'
+]
+
+// What tags do you want to use to select the instances?
+// Add the Tag Key and the Tag Value. Tag Value can be an input param.
+const searchedTags = [
+  { Key: "Environment", Value: env_tag_value }
+];
+
+// What aws accounts do you want to use?
+// Add the accounts you want Ellipisis to use. The label is used to
+// allow the user to choose what account to use.
+var accounts = [
+  {
+    label: "jack_acc",
+    key: ellipsis.env.JACK_AWS_KEY,
+    secret: ellipsis.env.JACK_AWS_SECRET
+  },
+  {
+    label: "ken_acc",
+    key: ellipsis.env.KEN_AWS_KEY,
+    secret: ellipsis.env.KEN_AWS_SECRET
```

Comment:
Can we change these environment variable names so that they seem less specific to certain people?